client/finality-grandpa/src/communication/mod.rs

```diff
@@ -48,8 +42,8 @@ use sp_runtime::traits::{Block as BlockT, Hash as HashT, Header as HeaderT, Numb
 use sc_telemetry::{telemetry, CONSENSUS_DEBUG, CONSENSUS_INFO};
 
 use crate::{
-	CatchUp, Commit, CommunicationIn, CommunicationOut, CompactCommit, Error,
-	Message, SignedMessage,
+	CatchUp, Commit, CommunicationIn, CommunicationOutH,
```

Comment:
What does the `H` in `CommunicationOutH` stand for?

Reply:
It's like `CommunicationOut` but "useful for forcing the hash to some type (e.g. `H256`) when the compiler can't do the inference.".